spec/models/manageiq/providers/kubevirt/infra_manager/provision/cloning_spec.rb

```diff
@@ -21,46 +21,24 @@
 
 describe ManageIQ::Providers::Kubevirt::InfraManager::Provision do
   context "Cloning" do
-    it "creates specific objects with pvc" do
+    it "calls clone on template" do
       source = FactoryGirl.create(:template_kubevirt)
       subject.source = source
       connection = double("connection")
 
-      allow(connection).to receive(:template).and_return(unprocessed_object("template.json"))
+      template = double("template")
+      allow(template).to receive(:name).and_return("example")
+      expect(template).to receive(:clone)
```

Comment:
if this expectation is supposed to be fulfilled by the last line of the test, I would move it to the line before last, I could not figure out if there is another expect missing in this spec.

Reply:
OK